cg/apps/demultiplex/sample_sheet/read_sample_sheet.py

```diff
@@ -34,7 +34,9 @@ def validate_samples_unique_per_lane(samples: list[FlowCellSample]) -> None:
         validate_samples_are_unique(samples=lane_samples)
 
 
-def get_sample_type_from_content(sample_sheet_content: list[list[str]]) -> Type[FlowCellSample]:
+def get_sample_type_from_content(
+    sample_sheet_content: list[list[str]],
+) -> Type[FlowCellSampleBcl2Fastq | FlowCellSampleBCLConvert]:
```

Comment:
Why is this changes? Are these not child classes?

Reply:
Just to make Pycharm not complain. Should I revert this? @Vince-janv

Reply:
What is Pycharm complaining about?

Reply:
Just type hint warnings

Reply:
resolved in person